php/Blockonomics.php

```diff
@@ -524,8 +525,7 @@ public function check_callback_secret($secret){
     }
 
     // Save the received payment info to the WooCommerce order
-    public function record_payment($value, $order, $wc_order){
-        update_post_meta($wc_order->get_id(), 'paid_'. $order['crypto'], $value/1.0e8);
+    public function record_transaction($value, $order, $wc_order){
```

Comment:
The code comment is still referring to the older. Both the names record_transaction adn record_payment are particularly similar, so we should attempt to clarify with correct code comments or more distinct function names